app/models/link.rb

```diff
@@ -7,4 +9,19 @@ class Link < ActiveRecord::Base
   scope :without_issue, ->() { where(issue_id: nil) }
 
   delegate :title, to: :issue, prefix: true, allow_nil: true
+
+  before_save :cleanup_link
+
+  PARAMS_TO_OMIT = %w[utm_source utm_medium]
```

Comment:
let's also reorganize file according to https://gist.github.com/rilian/8ca2ef7d52ca0181f0db